tox.ini

```diff
@@ -29,14 +29,16 @@ commands =
         {posargs:-n 8}
 
 [testenv:lint]
-deps =
-    -e.[dev]
 commands =
+    black . --diff --check
```

Comment:
Maybe add a `-q` here to supress the emoji messages

Reply:
I don't mind them. :stuck_out_tongue_winking_eye: